.github/workflows/generate-instance.yml

```diff
@@ -0,0 +1,50 @@
+name: Generate Cred Instance
+on:
+  # Trigger on merging to master.
+  push:
+    branches:
+      - master
+  # As well as every 6 hours.
+  schedule:
+    - cron: 0 */6 * * *
+
+jobs:
+  GenerateCredInstance:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false # Required to make github pages deployment work correctly
+
+      - name: Cache Data # Cache SourceCred Data, invalidating if any of the config changes or the SC version is updated
+        uses: actions/cache@v2
+        with:
+          path: '**/cache'
+          key: SC-${{ runner.os }}-${{ hashFiles('**/config.json', '**/sourcecred.json', '**/yarn.lock') }}
+
+      - name: Install Packages 🔧
+        run: yarn --frozen-lockfile
+
+      - name: Load Data into Graph 🔌
+        run: |
+          yarn load
+          yarn graph
+        env:
+          SOURCECRED_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SOURCECRED_DISCORD_TOKEN: ${{ secrets.SOURCECRED_DISCORD_TOKEN }}
+
+      - name: Compute Cred 🧮
+        run: yarn score
+
+      - name: Generate Frontend 🏗
+        run: |
+          yarn site
+          cp -r ./output ./site
+          cp ./sourcecred.json ./site
```

Comment:
Interesting -- I didn't realize that we could deploy from the folder named `site` -- taking this approach may let us clean up the site command so we don't emit symlinks. It may also enable us to start keeping the site/output in a separate git repository? This is something that @wchargin has been strongly advocating (keep the config in a separate git history from all the output data)

Reply:
@decentralion I think it's fine keeping it in the same repo, the way this action works is it deploys the github actions site on a separate branch (where the static site output is the only code). That branch will have a clean history separate from the master branch.